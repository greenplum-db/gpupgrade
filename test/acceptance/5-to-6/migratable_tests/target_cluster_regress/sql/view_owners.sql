-- Copyright (c) 2017-2023 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

--------------------------------------------------------------------------------
-- Create and setup migratable objects
--------------------------------------------------------------------------------

SELECT schemaname, viewname, viewowner
FROM pg_views
WHERE schemaname NOT IN ('pg_catalog', 'information_schema', 'gp_toolkit')
ORDER BY 1,2,3;
