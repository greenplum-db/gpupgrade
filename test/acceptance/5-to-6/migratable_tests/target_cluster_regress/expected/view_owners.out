-- Copyright (c) 2017-2023 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

--------------------------------------------------------------------------------
-- Create and setup migratable objects
--------------------------------------------------------------------------------

SELECT schemaname, viewname, viewowner FROM pg_views WHERE schemaname NOT IN ('pg_catalog', 'information_schema', 'gp_toolkit') ORDER BY 1,2,3;
 schemaname | viewname                 | viewowner  
------------+--------------------------+------------
 public     | preserve_view_owner_view | test_role1 
(1 row)
